README.md

```diff
@@ -1,14 +1,124 @@
 Reverse Tunneling Dialer
 ========================
 
-Client makes an outbound connection to a server.  The server can now do net.Dial from the
-server that will actually do a net.Dial on the client and pipe all bytes back and forth.
+remotedialer creates a two-way connection between two entities which allows
+services running on localhost or behind a NAT or firewall to be accessible by a
+remote server.
```

Comment:
I would retain the explicit mention of allowing the server to Dial via client sessions. The golang Dialer interface is well known. The proposed text makes it sound like it exposes the tunneled client directly to the server host, while in reality it just gives the server application a way to Dial through the client.

Reply:
done